data/hai/unfettered jobs.txt

```diff
@@ -335,6 +335,27 @@ mission "Unfettered Lead Asteroid Mining 2"
 		outfit "Lead" -70
 
 
+mission "Unfettered Titanium Asteroid Mining"
+	job
+	repeat
+	name "Gather Titanium from asteroids"
+	description "Mine and collect 50 tons of titanium, then bring the minerals to <destination> for <payment>. Lead is available in the asteroid field in Wah Yoot."
+	source
+		attributes "unfettered"
+		not planet "Warfeed"
+	destination
+		attributes "unfettered"
+		not planet "Warfeed"
+	to offer
+		random < 40
+		"cargo space" > 30
```

Comment:
Should this be 30 or 50?

Reply:
it's a bit below what you have available like the others